tests/datavolume_test.go

```diff
@@ -2644,9 +2658,12 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				Skip("Default storage class has no provisioner. The new storage class won't work")
 			}
 
-			By(fmt.Sprintf("verifying no storage class %s", testScName))
-			_, err := f.K8sClient.StorageV1().StorageClasses().Get(context.TODO(), scName, metav1.GetOptions{})
-			Expect(err).To(HaveOccurred())
+			var err error
+			if scName != "" {
+				By(fmt.Sprintf("verifying no storage class %s", scName))
+				_, err = f.K8sClient.StorageV1().StorageClasses().Get(context.TODO(), scName, metav1.GetOptions{})
+				Expect(err).To(HaveOccurred())
```

Comment:
probably want to be more specific here `Expect(err).To(MatchError(errors.IsNotFound, "k8serrors.IsNotFound"))`